deploy/olm-catalog/openshift-pipelines-operator/0.8.0/operator_v1alpha1_config_crd.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
why is the crd copied here? on purpose?

Reply:
yes, we use the `--update-crds flag` so each casv version gets bundled with the right CRD revision

in addition, in operatorhub repo they want nested bundle structure. This means newer CRDs can be shipped with each csv version without breaking older versions.
```
openshift-pipelines-operator
├── 0.5.2
│   ├── config.operator.tekton.dev.crd.yaml
│   └── openshift-pipelines-operator.v0.5.2.clusterserviceversion.yaml
├── 0.7.0
│   ├── config.operator.tekton.dev.crd.yaml
│   └── openshift-pipelines-operator.v0.7.0.clusterserviceversion.yaml
├── 0.8.0
│   ├── config.operator.tekton.dev.crd.yaml
│   └── openshift-pipelines-operator.v0.8.0.clusterserviceversion.yaml
└── openshift-pipelines-operator.package.yaml
```